cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/CustomJsonParser.java

```diff
@@ -1,10 +1,13 @@
 package com.sourcepoint.gdpr_cmplibrary;
 
+import android.app.MediaRouteActionProvider;
```

Comment:
maybe this dependency was added by mistake?

Reply:
@andresilveirah thanks for the note. Dependence cleared 🕺